content/javascript/concepts/strings/terms/padStart/padStart.md

```diff
@@ -0,0 +1,98 @@
+---
+Title: '.padStart()'
+Description: 'Takes an original string and returns a second string to the front in order to change the original string to a specified length.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Takes an original string and returns a second string to the front in order to change the original string to a specified length.
+
+## Syntax
+
+```pseudo
+originalString.padStart(targetLength, padString);
+```
+
+- `originalString` starting string value before padding.
+- `targetLength` indicates the length the resulting string should be.
+- `padString`, (optional) character or string value to pad the original string.
+
+## Examples
+
+
+Pad a string with spaces:
+
+```js
+const string = `Hi There!`;
+
+const paddedString = string.padStart(15);
+
+console.log(paddedString);
+// Output:       Hi There!
+```
+
+Pad a string with 0s:
+
+```js
+const numString = `4457`;
+
+const paddedNumString = numString.padStart(9, 0);
+
+console.log(paddedNumString);
+// Output: 000004457
+```
+
+Pad a string with a string less than the target length:
+
+```js
+const string = 'help!';
+const targetLength = 21;
+const padString = 'Can you '
+
+const paddedString = string.padStart(targetLength, padString);
+
+console.log(paddedString);
+// Output: Can you Can you help!
+```
+
+Pad a string with a string more than the target length:
+
+```js
+const string = 'help!';
+const targetLength = 9
+const padString = 'Can you '
+
+const paddedString = string.padStart(targetLength, padString);
+
+console.log(paddedString);
+// Output: Can help!
+```
```

Comment:
I would suggest putting all these use cases in one example, you can have multiple `console.log` showcasing multiple outputs.
After this bloc, use the ```shell``` block to show the output, avoid writing the output comments in the code itself. You can have comments in the code to explain the particular padding.